Sming/SmingCore/DateTime/DateTime.cpp

```diff
@@ -0,0 +1,416 @@
+/*
```

Comment:
Move `Sming/SmingCore/DateTime/DateTime.*` to `Sming/SmingCore/DateTime.*`. We don't need additional sub-directory for just one class.

Reply:
Fixed in working copy. Will push when coding style checked and committed. I am currently unable to remove the directory with git. Will investigate further.

Reply:
Try these commands:
```
git mv Sming/SmingCore/DateTime/DateTime.h Sming/SmingCore/DateTime.h
git mv Sming/SmingCore/DateTime/DateTime.cpp Sming/SmingCore/DateTime.cpp
git rm -r Sming/SmingCore/DateTime/
```

Reply:
I tried rm -r but git doesn't think the directory exists so maybe something I did removed it. It looks like it is now deleted.